changelogs/fragments/4457-support-datadog-monitors-type event-v2.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Ah, one thing: this is more a new feature than a bugfix, so it should be `minor_changes`.
```suggestion
minor_changes:
```

---

updated